packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -15,198 +15,52 @@ import '../../geometry.dart';
 import '../../ui/colors.dart';
 import '../../ui/theme.dart';
 import '../../utils.dart';
-import '../timeline_controller.dart';
-import '../timeline_model.dart';
+import 'timeline_controller.dart';
+import 'timeline_model.dart';
 
 class TimelineFlameChart extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final controller = Controllers.of(context).timeline;
     return LayoutBuilder(builder: (context, constraints) {
       return ValueListenableBuilder(
```

Comment:
having multiple nested ValueListenableBuilders is an anti-pattern. 
Either refactor to use a Stateful widget or create a 
ValueListenablesBuilder widget that takes a couple ValueListenables.

Reply:
done.